src/components/Modals/SendAssetModal.js

```diff
@@ -71,10 +74,12 @@ function SendAssetModal({
         assetData,
         remarkInfo: encodedRemark,
       };
-      if (isCongress) {
-        dispatch(congressActions.congressSendAssets.call({ ...data, executionBlock }));
-      } else {
-        dispatch(senateActions.senateSendAssets.call({ ...data, isCongress }));
+      if (officeType === OfficeType.CONGRESS) {
```

Comment:
and what if its just an user, sending assets through his wallet page, not an office at all?

Reply:
This is for office send assets, if he wants to send assets with remark, we will use TransferWithRemarkModal, which one handle any transaction with modal.

Reply:
If I remember correctly, when I was testing, user didnt have button to send assets, on the asset. This is a bug breaking current functionality, users should be able to click send on the asset